src/www/system_advanced_firewall.php

```diff
@@ -139,11 +139,7 @@
             unset($config['system']['lb_use_sticky']);
         }
 
-        if (!empty($pconfig['noantilockout'])) {
-            $config['system']['webgui']['noantilockout'] = true;
-        } elseif (isset($config['system']['webgui']['noantilockout'])) {
-            unset($config['system']['webgui']['noantilockout']);
-        }
+        $config['system']['webgui']['antilockout_interface'] = $pconfig['antilockout_interface'];
```

Comment:
historically these constructs exist to avoid PHP warnings and notices (check development mode)